src/particles/ImpactXParticleContainer.H

```diff
@@ -104,9 +104,19 @@ namespace impactx
                        amrex::Vector<amrex::ParticleReal> const & py,
                        amrex::Vector<amrex::ParticleReal> const & pz);
 
+        /** Set initial reference particle attributes
+         *
+         * @param refpart initial reference particle
```

Comment:
I think this is even more than `initial` since we will use the same method to overwrite the reference particle when we push it?
```suggestion
        /** Set reference particle attributes
         *
         * @param refpart reference particle
```